frontend/components/common/PrerenderedHTML.tsx

```diff
@@ -313,26 +342,38 @@ export const styles: SystemStyleObject = {
          paddingInline: 3,
       },
    },
+};
 
-   '.fa-svg-icon': {
-      svg: {
-         width: '1em',
-         height: '1em',
-         display: 'inline-block',
-
-         path: {
-            fill: 'currentColor',
-         },
-      },
-
-      '&.xs svg': {
-         width: '0.5em',
-         height: '0.5em',
-      },
-
-      '&.sm svg': {
-         width: '0.75em',
-         height: '0.75em',
-      },
-   },
+const commerceStyles: SystemStyleObject = {
+   ...sharedStyles,
 };
+
+export const PrerenderedHTML = chakra(function Prerendered({
+   html,
+   styles,
+   className,
+}: {
+   html: string;
+   styles: 'troubleshooting' | 'commerce';
+   className?: string;
+}) {
+   useEffect(() => {
+      import('lite-youtube-embed');
+   }, []);
+
+   const applyStyles = () => {
+      if (styles === 'troubleshooting') {
+         return troubleshootingStyles;
+      } else if (styles === 'commerce') {
+         return commerceStyles;
+      }
+   };
+
+   return (
+      <Box
+         className={`prerendered ${className}`}
+         sx={applyStyles()}
```

Comment:
> I am not the biggest fan of having to execute a function here. If there's a cleaner way to handle this, I'd love to hear it.

This is a little more verbose but I think it will work

```suggestion
   let sx;
   switch (styles) {
      case 'troubleshooting':
         sx = troubleshootingStyles;
         break;
      case 'commerce':
         sx = commerceStyles;
         break;
      default:
         assertNever(styles);
   }

   return (
      <Box
         className={`prerendered ${className}`}
         sx={sx}
```

Reply:
[`23ce020`](https://github.com/iFixit/react-commerce/pull/1946/commits/23ce020f1ffbda5d94867bae325cde90fbbc29a5)